modules/gapi/perf/cpu/gapi_imgproc_perf_tests_cpu.cpp

```diff
@@ -124,6 +124,25 @@ INSTANTIATE_TEST_CASE_P(SobelPerfTestCPU32F, SobelPerfTest,
         Values(1, 2),
         Values(cv::compile_args(IMGPROC_CPU))));
 
+INSTANTIATE_TEST_CASE_P(LaplacianPerfTestCPU, LaplacianPerfTest,
+                        Combine(Values(AbsExact().to_compare_f()),
+                                Values(CV_8UC1, CV_8UC3, CV_16SC1),
+                                Values(3, 5),
+                                Values(szVGA, sz720p, sz1080p),
+                                Values(-1, CV_16S, CV_32F),
+                                Values(cv::compile_args(IMGPROC_CPU))));
+
+INSTANTIATE_TEST_CASE_P(BilateralFilterPerfTestCPU, BilateralFilterPerfTest,
+                        Combine(Values(AbsExact().to_compare_f()),
+                                Values(CV_8UC1, CV_8UC3),
+                                Values(-1, CV_16S, CV_32F),
```

Comment:
Is it necessary to define this parameter with 3 different values? As I can see it's not used in the test body. Is it important to check the kernel's behaviour with different output initializations?

Reply:
Removed. Tests with parameters as in OCV.

Reply:
Actually, I meant only bilateral tests. As I can see, Laplacian uses the dtype argument as it can return back a Mat with types differ from input ones. Do they check this functionality in imgproc tests? If don't, then OK

Reply:
And, I'm sorry, should have mentioned: it is important to check almost all the functionality/range of parameters in our tests; but on the other hand there shouldn't be too many tests.. If you think all the checked parameters configuration is important, then leave them, otherwise drop them to reduce the testing time

Reply:
Removed single parameters from tests. Parameters in perf_tests are cut. Parameters in accuracy tests as in OCV.